tailwind/spacing.ts

```diff
@@ -0,0 +1,275 @@
+import type { Config } from "tailwindcss"
+
+/**
+ * Spacing for the tailwind config
+ * Provides scale in relative(rem) and absolute(px) units
+ *
+ * The pixel scale is used by default.
+ * All properties that do not explicitly set to use relative scale, use pixels.
+ *
+ * Relative scale is used for size definitions, like heights and widths.
+ */
+export const spacing = {
+  relativeSpacing: {
+    px: "1px",
+    0: "0px",
+    0.5: "0.125rem",
+    1: "0.25rem",
+    1.5: "0.375rem",
+    2: "0.5rem",
+    2.5: "0.625rem",
+    3: "0.75rem",
+    3.5: "0.875rem",
+    4: "1rem",
+    5: "1.25rem",
+    6: "1.5rem",
+    7: "1.75rem",
+    8: "2rem",
+    9: "2.25rem",
+    10: "2.5rem",
+    11: "2.75rem",
+    12: "3rem",
+    14: "3.5rem",
+    16: "4rem",
+    20: "5rem",
+    24: "6rem",
+    28: "7rem",
+    32: "8rem",
+    36: "9rem",
+    40: "10rem",
+    44: "11rem",
+    48: "12rem",
+    52: "13rem",
+    56: "14rem",
+    60: "15rem",
+    64: "16rem",
+    72: "18rem",
+    80: "20rem",
+    96: "24rem",
+  },
+  spacing: {
+    px: "1px",
+    0: "0px",
+    0.5: "2px",
+    1: "4px",
+    1.5: "6px",
+    2: "8px",
+    2.5: "10px",
+    3: "12px",
+    3.5: "14px",
+    4: "16px",
+    5: "20px",
+    6: "24px",
+    7: "28px",
+    8: "32px",
+    9: "36px",
+    10: "40px",
+    11: "44px",
+    12: "48px",
+    14: "56px",
+    16: "64px",
+    20: "80px",
+    24: "96px",
+    28: "112px",
+    32: "128px",
+    36: "144px",
+    40: "160px",
+    44: "176px",
+    48: "192px",
+    52: "208px",
+    56: "224px",
+    60: "240px",
+    64: "256px",
+    72: "288px",
+    80: "320px",
+    96: "384px",
+  },
+  flexBasis: ({ theme }) => ({
+    auto: "auto",
+    ...theme("relativeSpacing"),
+    "1/2": "50%",
```

Comment:
Can't we import this from tailwind's defaults instead of redefining it? The fewer custom code we have, the better.

Reply:
I couldn't find a way to do so. They have the exact same code, but with `theme("spacing")` helper. Do you have an idea how we can make it?

Reply:
Wouldn't just splatting `...theme('spacing')` just before make sure we keep the same keys and just override the next ones?

```tsc
...theme("spacing"),
...theme("relativeSpacing"),
```